GliaWidgets/Public/Glia/Glia+StartEngagement.swift

```diff
@@ -28,20 +28,25 @@ extension Glia {
         // `GliaError.engagementExists` is no longer thrown,
         // instead engagement is getting restored.
         guard let configuration = self.configuration else { throw GliaError.sdkIsNotConfigured }
-
+        self.interactor?.setQueuesIds(queueIds)
         // It is assumed that `features` to be provided from `configure` or via deprecated `startEngagement` method.
         let features = self.features ?? []
 
         if let engagement = environment.coreSdk.getCurrentEngagement() {
             if engagement.source == .callVisualizer {
                 throw GliaError.callVisualizerEngagementExists
             } else {
-                if interactor != nil, let rootCoordinator {
+                guard let interactor else {
+                    loggerPhase.logger.prefixed(Self.self).warning("Interactor is missing")
```

Comment:
Was logging events doc updated with this newly added log?

Reply:
Logging events have been added to the doc